www/_layouts/team.html

```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+team_tab: true
+---
+
+<div class="contribute-help">
```

Comment:
Mandatory to use this class? Does not seem fitting for the team page.

Reply:
I am testing this right now. Build takes ages.

Reply:
works